functions/Get-DbaQueryStoreConfig.ps1

```diff
@@ -106,8 +106,8 @@ Returns Query Store configuration settings for every database on the ServerA\sql
 				}
 				
 				[pscustomobject]@{
-					Instance = $instance.name
-					Database = $db.name
+					Instance = $instance
```

Comment:
So within verbose statements (or, formerly, output), this is okay but on output, I'd prefer the cleaned up .name output.